### PR TITLE
[DOCU-2925] HTTPie cleanout

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

Cleaning out HTTPie from our docs, as its an extra tool that someone would need to install. This causes a problem for customers.

* Converting any httpie examples into curl
* For curl/httpie navtabs, removing the tabs and leaving curl only
* Cleaned up and tested the ["Enable RBAC"](https://docs.konghq.com/gateway/latest/production/access-control/enable-rbac/) doc. It became apparent that this needed doing as I couldn't just convert inaccurate commands from httpie to curl, I had to make sure they were real. This doc has also commonly been a source of bug reports in the past, and we've just never gotten to it.

Issue reported on Slack: https://kongstrong.slack.com/archives/CDSTDSG9J/p1685430807824339

https://konghq.atlassian.net/browse/DOCU-2925
https://konghq.atlassian.net/browse/DOCU-288

### Testing instructions

Netlify link: Lots of pages, but here are the main ones:

* https://deploy-preview-5652--kongdocs.netlify.app/gateway/latest/production/access-control/enable-rbac/ - big page update, needs the most review.

Removed/converted HTTPie:
* https://deploy-preview-5652--kongdocs.netlify.app/gateway/latest/kong-enterprise/workspaces/
* https://deploy-preview-5652--kongdocs.netlify.app/gateway/latest/kong-enterprise/consumer-groups/
* https://deploy-preview-5652--kongdocs.netlify.app/gateway/latest/kong-enterprise/event-hooks/
* https://deploy-preview-5652--kongdocs.netlify.app/gateway/latest/kong-enterprise/plugin-ordering/get-started/
* https://deploy-preview-5652--kongdocs.netlify.app/gateway/latest/production/access-control/register-admin-api/
* https://deploy-preview-5652--kongdocs.netlify.app/gateway/latest/reference/key-management/

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 1]
```diff
@@ -9,612 +9,648 @@ use case demonstrates how **RBAC with workspaces** can be coupled
...
 
-## Bootstrapping the first RBAC user—the Super Admin
+To sum up, workspaces and RBAC are complementary: workspaces provide segmentation of
+Admin API entities, while RBAC provides access control.
 
-**Note:** It is possible to create the first Super Admin at the time
-of migration as described in the [Getting Started Guide][getting-started-guide].
-If you chose this option, skip to [Enforcing RBAC](#enforcing-rbac).
+{:.note}
+> **Note:** The example responses in this guide are often partial excerpts
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:27]:
```suggestion
> **Note:** The example responses in this guide are often excerpts of full responses,
```

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 2]
```diff
@@ -9,612 +9,648 @@ use case demonstrates how **RBAC with workspaces** can be coupled
...
-```
+{:.note}
+> It's possible to create the first super admin at installation time.
+If you chose this option, skip to [Enforcing RBAC](#enforcing-rbac).
 
-As the `super-admin` user name coincides with an existing `super-admin`
-role, it gets automatically added to the `super-admin` role—which can be
-confirmed with the following command:
+{{site.base_gateway}} ships with a set of default RBAC roles: `super-admin`,
+the `admin`, and `read-only`, which the task of creating a Super Admin user easy:
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:43]:
```suggestion
{{site.base_gateway}} ships with a set of default RBAC roles: `super-admin`,
the `admin`, and `read-only`, which makes the task of creating a super admin user easy:
```

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 3]
```diff
@@ -9,612 +9,648 @@ use case demonstrates how **RBAC with workspaces** can be coupled
...
+      }
+    }
+    ```
 
 ## Enforcing RBAC
 
-As the `super-admin` user has just been created, the Kong Admin may now
-restart Kong with RBAC enforced, with, e.g.:
+As the `super-admin` user has just been created, the Kong admin may now
+restart Kong with RBAC enforced:
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:97]:
```suggestion
With the `super-admin` user created, the Kong admin can now restart 
{{site.base_gateway}} with RBAC enforced:
```

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 4]
```diff
@@ -9,612 +9,648 @@ use case demonstrates how **RBAC with workspaces** can be coupled
...
 
 ```
 KONG_ENFORCE_RBAC=on kong restart
 ```
 
-**NOTE**: This is one of the possible ways of enforcing RBAC and restarting
-Kong; another possibility is editing the Kong configuration file and
+This is one of the possible ways of enforcing RBAC and restarting
+Kong. Another possibility is editing the Kong configuration file and
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:104]:
```suggestion
Kong. Another options is editing the {{site.base_gateway}} configuration file and
```

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 5]
```diff
@@ -9,612 +9,648 @@ use case demonstrates how **RBAC with workspaces** can be coupled
...
+      "created_at": 1531015221000,
+      "name": "adminB"
     }
-  ]
-}
-```
+    ```
 
-Similarly, workspaces teamB and teamC only show their respective admins:
+1. With this, both of the teams have one admin and each admin can only be seen
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:201]:
```suggestion
1. Both of the teams now have one admin and each admin can only be seen
```

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 6]
```diff
@@ -9,612 +9,648 @@ use case demonstrates how **RBAC with workspaces** can be coupled
...
-## Super Admin Creates Admin Roles for Teams
+## Super admin creates admin roles for teams
 
-Super Admin is now done creating RBAC Admin users for each team; his next
+The super admin is now done creating RBAC admin users for each team. The next
 task is to create admin roles that will effectively grant permissions to admin
 users.
 
-The admin role must have access to all of the Admin API, restricted to his
+The admin role must have access to all of the Admin API, restricted to their
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:254]:
```suggestion
The admin role must have access to all of the Admin API, restricted to its
```

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 7]
```diff
@@ -9,612 +9,648 @@ use case demonstrates how **RBAC with workspaces** can be coupled
...
+          "enabled": true,
+          "id": "bca4e390-fbbf-4a46-b55d-f4642efc14bb",
+          "name": "adminA",
+          "user_token": "$2b$09$oLyKTIDuKriPZ.SD5wYtxeMclGYNDn4udJkQG0NGx/Aq3j9j/tWsa",
+          "user_token_ident": "0ebb5"
+            }
+    }
+    ```
+
+1. With these steps, Team A's admin user is now able to manage their team. To
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:338]:
```suggestion
1. Team A's admin user is now able to manage their team. To
```

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 8]
```diff
@@ -9,612 +9,648 @@ use case demonstrates how **RBAC with workspaces** can be coupled
...
+      "message": "Invalid RBAC credentials"
+    }
+    ```
 
-```
-http :8001/teamA/rbac/users Kong-Admin-Token:n5bhjgv0speXp4N7rSUzUj8PGnl3F5eG
-{
-  "total": 1,
-  "data": [
+1. Now try to list RBAC users in Team A's workspace:
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:354]:
```suggestion
1. Now try listing RBAC users in Team A's workspace:
```

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 9]
```diff
@@ -9,612 +9,648 @@ use case demonstrates how **RBAC with workspaces** can be coupled
...
 
-If the same procedure is repeated for Team B and Team C, they will end up with
+If the same procedure is repeated for Team B, they will end up with
 a similar set up, with an admin role and an admin user, both restricted to the
 team's workspace.
 
-And so Super Admin ends his participation; individual team admins are now able
-to set up his teams users and entities!
+The super admin's job is now done. 
+Individual team admins are now able to set up their teams users and entities!
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:382]:
```suggestion
Individual team admins are now able to set up their team's users and entities!
```

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 10]
```diff
@@ -9,612 +9,648 @@ use case demonstrates how **RBAC with workspaces** can be coupled
...
+step is to create team users, such as engineers that are part of Team A or B. 
+Let's go ahead and do that, using Admin A's user token.
 
 Before regular users can be created, a role needs to be available for them.
-Such a role needs to have permissions to all of Admin API endpoints, except
-RBAC and Workspaces—regular users will not need access to these in general
-and, if they do, the Admin can grant them.
+This role needs to have permissions to all of Admin API endpoints, except
+RBAC and workspaces. Regular users don't need access to these endpoints in general
+and if they do, the admin can grant them.
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:393]:
```suggestion
This role needs to have permissions to all of the Admin API endpoints, except
RBAC and workspaces. Regular users don't need access to these endpoints,
and if they do, the admin can grant them individually.
```

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 11]
```diff
@@ -9,612 +9,648 @@ use case demonstrates how **RBAC with workspaces** can be coupled
...
+    ```
 
-```
-http :8001/teamA/rbac/users/foogineer/roles roles=users Kong-Admin-Token:n5bhjgv0speXp4N7rSUzUj8PGnl3F5eG
-{
-  "roles": [
+{:.important}
+> **Important**: As explained in the [Wildcards in Permissions](#wildcards-in-permissions)
+section, the meaning of `*` is not the expected generic globbing one might
+be used to. As such, `/rbac/*` or `/workspaces/*` do not match all of the
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:500]:
```suggestion
> **Important**: As explained in the [Wildcards in Permissions](#wildcards-in-permissions)
section, the meaning of `*` is not the same as generic globbing.
 As such, `/rbac/*` or `/workspaces/*` do not match all of the
```

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 12]
```diff
@@ -9,612 +9,648 @@ use case demonstrates how **RBAC with workspaces** can be coupled
...
+RBAC and Workspaces endpoints. For example, to cover all of the RBAC API,
+one would have to define permissions for the following endpoints:
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:502]:
```suggestion
RBAC and Workspaces endpoints. For example, to cover all of the RBAC API,
you would have to define permissions for the following endpoints:
```

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 13]
```diff
@@ -627,322 +663,300 @@ http :8001/teamA/plugins Kong-Admin-Token:dNeYvYAwvjOJdoReVJZXF8vLBXQioKkI
...
+
+In addition to endpoint permissions, RBAC in {{site.base_gateway}} supports entity-level
 permissions, meaning that particular entities, identified by their unique ID,
 can be allowed or disallowed access in a role.
 
-Refreshing our minds, RBAC is [enforced](#enforcing-rbac) with the `enforce_rbac`
-configuration directive—or with its `KONG_ENFORCE_RBAC` environment variable
+RBAC is [enforced](#enforcing-rbac) with the `enforce_rbac`
+configuration directive, or with its `KONG_ENFORCE_RBAC` environment variable
 counterpart. Such directive is an enum, with 4 possible values:
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:713]:
```suggestion
counterpart. The directive is an enum, with the following possible values:
```

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 14]
```diff
@@ -627,322 +663,300 @@ http :8001/teamA/plugins Kong-Admin-Token:dNeYvYAwvjOJdoReVJZXF8vLBXQioKkI
...
-access control
-- `entity`: applies **only** Entity-level access control
-- `both`: applies **both Endpoint and Entity level access control**
+- `on`: Applies endpoint-levelaccess control
+- `entity`: Applies **only** Entity-level access control
+- `both`: Applies **both Endpoint and Entity level access control**
 - `off`: disables RBAC enforcement
 
-If one sets it to either `entity` or `both`, Kong will enforce entity-level
+If set to `entity` or `both`, Kong will enforce entity-level
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:720]:
```suggestion
If set to `entity` or `both`, Kong enforces entity-level
```

File: app/_src/gateway/production/access-control/enable-rbac.md [hunk 15]
```diff
@@ -627,322 +663,300 @@ http :8001/teamA/plugins Kong-Admin-Token:dNeYvYAwvjOJdoReVJZXF8vLBXQioKkI
...
-  "port": 80,
-  "path": null,
-  "updated_at": 1531066074,
-  "retries": 5,
-  "write_timeout": 60000
-}
-```
+Let's assume that Admin A [enabled entity-level enforcement](#enforcing-rbac) as well.
+Note that as `qux` has **no endpoint-level permissions**. If both endpoint and
+entity-level enforcement is enabled, `qux` won't  be able to read their entities, as
```
Comment (lena-larionova) [inline app/_src/gateway/production/access-control/enable-rbac.md:867]:
```suggestion
entity-level enforcement is enabled, `qux` won't be able to read their entities, as
```

**Review by lena-larionova — COMMENTED** [review]
Editing the RBAC topic further.

**Review by lena-larionova — COMMENTED** [review]
Editing the RBAC topic further.

**Review by Guaris — APPROVED** [review]
Small issue I found. Approving it and letting you merge.

**Pull request merged**